scripts/collabrator-check.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python3
+
+import subprocess
+import sys
+
+base_git_sha=sys.argv[1]
+branch_git_sha=sys.argv[2]
+
+# print(base_git_sha[0:7])
+# print(branch_git_sha[0:7])
+
+# Get the merge diff results using the git sha of the base branch and the new branch. The filter checks for files that have
+# been added, modified, etc and filters out deleted changes. It will put the result into a pipe to be used with grep below.
+ps = subprocess.Popen(('git', 'diff' , '-U0', '--diff-filter=ACMRT', base_git_sha[0:7], branch_git_sha[0:7]), stdout=subprocess.PIPE)
+ps.wait()
+# The diff will return a string value if there were changes.
+if ps.stdout != "":
+    # Need a try exception block here because grep returns a negative result if it doesn't find the expected value.
+    try:
+        # Look for the 'added_by' line in the diff results
+        grep_output = subprocess.check_output(('grep', '+      added_by'), stdin=ps.stdout, text=True)
+    except:
+        # Grep didn't find an 'added_by' line, check finished.
+        print ("Check N/A")
+        sys.exit(0)
+    else:
+
+        print(grep_output)
+
+        # Some .tf files have this comment, so remove it at the start of the check
+        grep_output = grep_output.replace("awesome.team@digital.justice.gov.uk", "abc")
+
+        # Count the number of email @ symbols in the diff output
+        num_email_symbols = grep_output.count('@')
+        # print("There are ", num_email_symbols, " @ in the changes")
```

Comment:
We could clean up the commented out print statements if they're no longer needed.